src/tests/issues.targets

```diff
@@ -590,6 +590,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/GC/Scenarios/DoublinkList/dlstack/*">
             <Issue>Release only crash</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/HardwareIntrinsics/General/Vector256/*">
```

Comment:
Shouldn't this be `Vector256/**` - double star?

Reply:
Thanks!